### ISSUE TITLE
Unintuitive weapon sorting

### ISSUE THREAD

**Issue by nstringham** [SkyCryptWebsite/SkyCrypt#678]
**Describe the bug**
weapons are not in an intuitive order

**Profile/Location**
any but let's use this for an example https://sky.shiiyu.moe/stats/Cookie_Wookie_7/Papaya#Weapons

**Expected behavior**
the Tactician's Sword should come before the Hurricane Bow because the Hurricane Bow is in a backpack

**Actual behavior**
the Hurricane Bow comes before the Tactician's Sword because the Hurricane Bow is in slot `2` of the backpack but the Tactician's Sword is in slot `4` of the inventory

**summary**
basically, I think items should be sorted by what container they are in before what part of the container they are in

**Comment by lochyj** [comment]
It's an easy fix i think but it looks kinda funny if it isn't sorted by rarity / color.
It sorts from inventory -> ender-chest -> storage
To change this go to `lib.js L1111` and remove `.sort(itemSorter)`

**By rarity:**
![image](https://user-images.githubusercontent.com/79903111/183284951-c6c50716-6df0-4a59-bb29-6c81e168c083.png)
**By location in users containers:**
![image](https://user-images.githubusercontent.com/79903111/183284922-0a93bbe1-690e-46c8-b6ef-1023f5372212.png)

**Comment by DuckySoLucky** [comment]
@metalcupcake5
